packages/zoe/test/swingsetTests/upgradeCoveredCall/coveredCall-durable-V3.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+
+import { M, fit } from '@agoric/store';
+import '../../../exported.js';
+
+import { vivifyKind, vivifySingleton } from '@agoric/vat-data';
+import { swapExact } from '../../../src/contractSupport/index.js';
+import { isAfterDeadlineExitRule } from '../../../src/typeGuards.js';
+
+const { details: X } = assert;
+
+const sellSeatExpiredMsg = 'The covered call option is expired.';
+
+/**
+ * @see original version in .../zoe/src/contracts/coveredCall.js and upgradeable
+ * version in contracts/coveredCall-durable.js.
+ *
+ * This variant has minor changes to the returned strings that make it
+ * identifiable, to demonstrate that upgrade has occurred.
+ *
+ * @param {ZCF} zcf
+ * @param {unknown} _privateArgs
+ * @param {import('@agoric/vat-data').Baggage} instanceBaggage
+ */
+const vivify = async (zcf, _privateArgs, instanceBaggage) => {
```

Comment:
shouldn't the necessary param come earlier? (`instanceBaggage` is always necessary in `vivify` and `privateArgs` is not).

Conceptually I think of `instanceBaggage` a bit like `self` in Python so I'd suggest putting it first,
```suggestion
const vivify = async (instanceBaggage, zcf) => {
```

Reply:
It's a good idea, but we're trying to maintain backward compatibility, and continue to support non-durable contracts as first-class citizens, so we can't go this direction.

Reply:
I see @turadg 's thumbs up. Resolving.